node/network/availability-distribution/Cargo.toml

```diff
@@ -7,16 +7,16 @@ edition = "2018"
 [dependencies]
 futures = "0.3.15"
 tracing = "0.1.26"
-parity-scale-codec = { version = "2.0.0", features = ["std"]  }
+parity-scale-codec = { version = "2.0.0", features = ["std"] }
 polkadot-primitives = { path = "../../../primitives" }
 polkadot-erasure-coding = { path = "../../../erasure-coding" }
 polkadot-subsystem = { package = "polkadot-node-subsystem", path = "../../subsystem" }
 polkadot-node-network-protocol = { path = "../../network/protocol" }
 polkadot-node-subsystem-util = { path = "../../subsystem-util" }
 polkadot-node-primitives = { path = "../../primitives" }
-polkadot-node-core-runtime-api = { path = "../../core/runtime-api" }
-sp-application-crypto = { git = "https://github.com/paritytech/substrate", branch = "master" }
-sp-core = { git = "https://github.com/paritytech/substrate", branch = "master", features = ["std"]  }
+sp-core = { git = "https://github.com/paritytech/substrate", branch = "master", features = [
+    "std",
+] }
```

Comment:
nit: single item feature lists don't necessarily need linebreaks, is there a way to teach the toml formatter that?

Reply:
There is but that's a whole new PR. For now I've manually put them back to what they were.